### PR TITLE
fix(internal): Pillow instead of imageio

### PR DESCRIPTION
**Pull request by aarnphm**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: bentoml/_internal/utils/formparser.py [hunk 1]
```diff
@@ -159,10 +159,14 @@ async def populate_multipart_requests(request: Request) -> t.Dict[str, Request]:
...
 
-async def concat_to_multipart_responses(responses: _ResponseList) -> StreamingResponse:
-    resp = io.BytesIO()
-    for _resp in [req_[1] for req_ in responses]:
-        resp.write(_resp.body)
+async def concat_to_multipart_responses(responses: _ResponseList) -> Response:
     boundary = binascii.hexlify(os.urandom(16)).decode("ascii")
     headers = {"content-type": f"multipart/form-data; boundary={boundary}"}
-    return StreamingResponse(resp, headers=headers)
+    body = (
```
Comment (aarnphm) [inline bentoml/_internal/utils/formparser.py:165]: @bojiang RFC1341 compatible

File: bentoml/_internal/io_descriptors/image.py [hunk 2]
```diff
@@ -116,16 +117,12 @@ async def from_http_request(self, request: Request) -> "PIL.Image":
     async def to_http_response(
         self, obj: t.Union["np.ndarray", "PIL.Image.Image"]
     ) -> Response:
-        if isinstance(obj, np.ndarray):
-            pass
-        elif isinstance(obj, PIL.Image.Image):
-            obj = np.array(obj)
-        else:
+        if not any(isinstance(obj, i) for i in [np.ndarray, PIL.Image.Image]):
```
Comment (bojiang) [inline bentoml/_internal/io_descriptors/image.py:120]: why insist this lol

File: bentoml/_internal/io_descriptors/image.py [hunk 3]
```diff
@@ -116,16 +117,12 @@ async def from_http_request(self, request: Request) -> "PIL.Image":
     async def to_http_response(
         self, obj: t.Union["np.ndarray", "PIL.Image.Image"]
     ) -> Response:
-        if isinstance(obj, np.ndarray):
-            pass
-        elif isinstance(obj, PIL.Image.Image):
-            obj = np.array(obj)
-        else:
+        if not isinstance(obj, [np.ndarray, PIL.Image.Image]):
```
Comment (parano) [inline bentoml/_internal/io_descriptors/image.py:120]: this is causing an error in tests:
```
TypeError: isinstance() arg 2 must be a type or tuple of types
```
let's make sure lint and existing tests all pass before merging going forward

File: bentoml/_internal/io_descriptors/image.py [hunk 4]
```diff
@@ -116,16 +117,12 @@ async def from_http_request(self, request: Request) -> "PIL.Image":
...
             raise InvalidArgument(
-                f"Unsupported Image type received: {type(obj)}, `{self.__class__.__name__}`"
-                " supports only `np.ndarray` and `PIL.Image`"
+                f"Unsupported Image type received: {type(obj)}, `{self.__class__.__name__}` supports only `np.ndarray` and `PIL.Image`"
             )
+        image = PIL.Image.fromarray(obj) if isinstance(obj, np.ndarray) else obj
 
         ret = io.BytesIO()
-        imageio.imsave(ret, im=t.cast(t.Type["np.ndarray"], obj), format=self._ext)
+        image.save(ret, self._ext)
```
Comment (parano) [inline bentoml/_internal/io_descriptors/image.py:127]: this is causing an error `KeyError: 'JPG'`, see https://github.com/bentoml/BentoML/runs/4151602877?check_suite_focus=true#logs

fixed it here: https://github.com/bentoml/BentoML/pull/1979/commits/7cf5ba4257e79933ca34a6038d80849b46235e4c

**Pull request merged**